.readme-partials.yml

```diff
@@ -25,3 +25,29 @@ body: |-
 
   The `@google-cloud/logging` library will handle batching and dispatching
   these log lines to the API.
+
+  ## Writing to Stdout
+
+  Serverless applications like Cloud Functions, Cloud Run, and App Engine
+  should use the `LogSync` class to synchronously log to `stdout` or another
```

Comment:
While this is true for serverless, this is also true for GKE, GCE and other platforms. By default we want folks to use stdout logging and let the agent pick it up - unless a particular scenario requires sending logs directly to Cloud Logging (e.g. hybrid, transitions between one logging vendor to another, etc) Can you rephrase this a bit?

Reply:
ok!